app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -1,11 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<defaultsMap>
-  <entry>
-    <key>proximity_enabled</key>
-    <value>false</value>
-  </entry>
-  <entry>
-    <key>contest_goal</key>
-    <value>11</value>
-  </entry>
-</defaultsMap>
+<defaultsMap />
```

Comment:
do we need an empty map?

Reply:
It's just empty for now really, will be restored (albeit not for the London 2017 milestone). I don't want to remove it yet as we'll need it "soon"